container_linux_test.go

```diff
@@ -175,3 +176,79 @@ func TestShimInCgroup(t *testing.T) {
 
 	<-statusC
 }
+
+func TestContainerHostname(t *testing.T) {
```

Comment:
It should if TCP/IP is enabled, which it should be. I wanted to test, but someone broke my azure vm; I need to setup a new one.